src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
+
+        Ball ball = new Ball();
+
+        ball.setBallColor(ColorSupplier.getRandomColor());
```

Comment:
Since getRandomColor() isn't static, you can make ColorSupplier class-level variable. Check [common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#think-about-which-variables-should-be-local-in-the-method-and-which-should-be-class-level)
```suggestion
        ball.setBallColor(colorSupplier.getRandomColor());
```